src/redteam/index.ts

```diff
@@ -252,16 +255,46 @@ export async function synthesize({
       );
       logger.debug(`Added ${pluginTests.length} ${plugin.id} test cases`);
       pluginResults[plugin.id] = { requested: plugin.numTests, generated: pluginTests.length };
+    } else if (plugin.id.startsWith('file://')) {
+      logger.debug(`Loading custom plugin from ${plugin.id}`);
+      if (plugin.id.endsWith('.yaml')) {
+        const filePath = plugin.id.slice('file://'.length);
+
+        const definition = yaml.load(fs.readFileSync(filePath, 'utf8')) as {
+          generator: string;
```

Comment:
nit, you may want to support an id field in addition to generator / grader so the id is not just the file path

Reply:
going to skip this because I think the only place this would have an effect is the table outputted by `generate`?